### ISSUE TITLE
Lambdas

### ISSUE THREAD

**Issue by gorbak25** [erlscripten/erlscripten#7]
```
[INFO] Transpiling aeb_disassemble
Erlscripten encountered a fatal error while compiling /home/gorbak25/aebytecode/_build/local/lib/aebytecode/ebin//aeb_disassemble.beam
Error: error
Reason: {unimplemented_expr,
            {'fun',20,
                {function,{atom,20,io},{atom,20,format},{integer,20,2}}}}
Stacktrace: [{erlps_transpiler,transpile_expr,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,888}]},
             {erlps_transpiler,transpile_exprs,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,893}]},
             {erlps_transpiler,transpile_exprs,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,894}]},
             {erlps_transpiler,transpile_expr,3,

```

**Comment by gorbak25** [comment]
```
[INFO] Transpiling shell_docs
Erlscripten encountered a fatal error while compiling /usr/lib/erlang/lib/stdlib-3.13.2/ebin/shell_docs.beam
Error: error
Reason: {unimplemented_expr,
         {'fun',74,
          {clauses,
           [{clause,74,
             [{var,74,'Elem'}],
             [],
             [{op,74,'orelse',
               {op,74,'=:=',{var,74,'Elem'},{atom,74,a}},
               {op,74,'orelse',
                {op,74,'=:=',{var,74,'Elem'},{atom,74,code}},
                {op,74,'orelse',
                 {op,74,'=:=',{var,74,'Elem'},{atom,74,i}},
                 {op,74,'=:=',{var,74,'Elem'},{atom,74,em}}}}}]}]}}}
Stacktrace: [{erlps_transpiler,transpile_expr,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,888}]},
             {erlps_transpiler,transpile_exprs,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,893}]},
             {erlps_transpiler,transpile_expr,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,780}]},
             {erlps_transpiler,transpile_expr,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,696}]},
             {erlps_transpiler,transpile_body,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,590}]},
             {erlps_transpiler,transpile_body,2,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,575}]},
             {erlps_transpiler,transpile_function_clause,2,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,238}]},
             {erlps_transpiler,'-transpile_function/2-lc$^0/1-0-',2,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,121}]}]

```

**Comment by gorbak25** [comment]
```

[INFO] Transpiling erl_lint
Erlscripten encountered a fatal error while compiling /usr/lib/erlang/lib/stdlib-3.13.2/ebin/erl_lint.beam
Error: error
Reason: {unimplemented_expr,
            {'fun',49,
                {clauses,
                    [{clause,49,
                         [{var,49,'Opt'},{var,49,'_Def'}],
                         [[{op,49,'=:=',{var,49,'Opt'},{var,49,'On'}}]],
                         [{atom,49,true}]},
                     {clause,50,
                         [{var,50,'Opt'},{var,50,'_Def'}],
                         [[{op,50,'=:=',{var,50,'Opt'},{var,50,'Off'}}]],
                         [{atom,50,false}]},
                     {clause,51,
                         [{var,51,'_Opt'},{var,51,'Def'}],
                         [],
                         [{var,51,'Def'}]}]}}}
Stacktrace: [{erlps_transpiler,transpile_expr,3,
                               [{file,"/home/gorbak25/erlscripten/src/erlps_transpiler.erl"},
                                {line,888}]},

```

**Comment by gorbak25** [comment]
Fixed by c02aa84a9c1174f506b5b86a27525d6c82e65010
